sormas-backend/src/test/java/de/symeda/sormas/backend/labmessage/LabMessageFacadeEjbTest.java

```diff
@@ -33,4 +39,17 @@ public void deleteLabMessage() {
 
 		assertThat(getLabMessageFacade().getByUuid(labMessage.getUuid()), is(nullValue()));
 	}
+
+	@Test
+	public void testGetByReportId() {
```

Comment:
I would like the test to be a little more extensive:
What happens when null or an empty string is given as reportId? What happens with reportId with a space " " at the end or beginning? Is in those cases really the correct set of LabMessageDtos returned? 

Is there a distinction between uppercase and lowercase letters? I think there should not be, at least in some other parts of SORMAS, there is none (e.g. [here](https://github.com/hzi-braunschweig/SORMAS-Project/blob/6d3a033031eae61ba09c75f7f70ebc15e5528f64/sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java#L3653), which also is about external Ids of some sort).